.github/workflows/pr-create.yml

```diff
@@ -0,0 +1,41 @@
+name: TNP PR Created
+
+on:
+  pull_request:
+    branches:
+      - release-*
```

Comment:
Don't we want `master` to be in this list as well?

Reply:
Yeah, that'd be a good idea. I updated it but would it be better if we included **all** pull requests from every branch?

Reply:
Hmmm.... that's actually a great point. As long as we're judicious about what branches we merge to, then maybe this restriction is pointless?